### DISCUSSION TITLE
App windows overview (à la Unity).

### DISCUSSION THREAD

**Discussion by Blast-City** [elementary/gala#985] [Ideas]
Right now there is two types of windows overview:

Super+W - Windows on selected workspace.
Super+A - Windows across all workspaces.

What if there was a third one that showed all the open windows of an app across all workspaces. This overview would be accessed when clicking on a focused app dock icon if this app contains multiple windows (Like Unity).

The reason for this is that scrolling on the app dock icon only works if there is two open windows. More than that and it starts to get unpredictable. If there is fullscreen windows in the middle is simply impossible.

This would substitute the dock mouse scroll and the window list on the icon context menu.

Would this be interesting?

**Comment by Blast-City** [comment]
Can anyone close this discussion? elementary/dock#85 and what it means for the future can solve this issue.

**Comment by hanaral** [comment]
The current window and workspace overview looks like its going away - but possibly there should be a way to quickly see all open apps so that the user can drag and drop between overlapping windows too.
